crates/sui-types/src/sui_system_state/simtest_sui_system_state_inner.rs

```diff
@@ -227,6 +227,7 @@ impl SuiSystemStateTrait for SimTestSuiSystemStateInnerV1 {
                         narwhal_primary_address: metadata.primary_address.clone(),
                         narwhal_worker_address: metadata.worker_address.clone(),
                         voting_power: validator.voting_power,
+                        hostname: "".to_string(),
```

Comment:
Is this the right value? Feels strange to me that it should just be empty

Reply:
Is this field required as part of protocol to run the system? If so, we will need to propagate it properly from their name.
But in general, I am curious why do we have to introduce a separate hostname field, instead of just using their public key?

Reply:
Well the simtest validator type doesn't have name. But I guess it depends on how the hostname is used. If it's just used for metric/logging that we don't care and "" is probably fine (but worth a comment)